### PR TITLE
Attempt fixing failing windows jasmine build

### PR DESCRIPTION
**Pull request by GaneshSPatil**
Description:

Attempt fixing failing windows jasmine build

### CODE REVIEW

File: server/src/main/webapp/WEB-INF/rails/webpack/views/pages/pipeline_activity/spec/comment_widget_spec.tsx [hunk 1]
```diff
@@ -207,7 +208,7 @@ describe("CommentWidget", () => {
       helper.clickByTestId("edit-comment-button", modal);
 
       expect(helper.byTestId("textarea-to-add-or-edit-comment", modal)).toBeInDOM();
-      expect((helper.byTestId("textarea-to-add-or-edit-comment", modal) as HTMLInputElement).value).toBe(comment());
+      expect($("textarea").val()).toBe(comment());
```
Comment (bdpiparva) [inline server/src/main/webapp/WEB-INF/rails/webpack/views/pages/pipeline_activity/spec/comment_widget_spec.tsx]: Jquery??. I thought we are going away from it.

Comment (GaneshSPatil) [reply server/src/main/webapp/WEB-INF/rails/webpack/views/pages/pipeline_activity/spec/comment_widget_spec.tsx]: Fixed!!

**Comment by GaneshSPatil** [comment]
Jasmine job passed: https://build.gocd.org/go/tab/build/detail/build-windows-PR/5432/build-non-server/1/jasmine

File: server/src/main/webapp/WEB-INF/rails/webpack/views/pages/pipeline_activity/spec/comment_widget_spec.tsx [hunk 2]
```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import $ from "jquery";
```
Comment (GaneshSPatil) [inline server/src/main/webapp/WEB-INF/rails/webpack/views/pages/pipeline_activity/spec/comment_widget_spec.tsx]: unused import 🤦‍♂

**Pull request merged**